frontend/src/views/LoginView.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <div>
+    <h1>LOGIN</h1>
+    <form
+      @submit.prevent="login"
+      style="border: solid 1px black; width: 25vw; min-width: 300px"
+      class="mx-auto p-3"
+    >
+      <div class="bg-danger text-white p-1 mx-4 rounded" v-if="erreurs.length">
+        <span v-for="erreur in erreurs" :key="erreur">{{ erreur }}</span>
+      </div>
+      <input
+        class="text-center p-1 my-3 rounded"
+        style="background-color: #e3e3e3"
+        v-model="email"
+        placeholder="Courriel"
+      />
+      <div style="width: 100%"></div>
+      <input
+        class="text-center p-1 rounded"
+        style="background-color: #e3e3e3"
+        v-model="motdepasse"
+        placeholder="Mot de passe"
+        type="password"
+      />
+      <div style="width: 100%"></div>
```

Comment:
Pretty much the same as signup around here, juste make sure not to use the 'style' attribute inside html and use bootstrap classes like you're currently doing !